### PR TITLE
Fixes to smartfees.py

### PR DESCRIPTION
**Pull request by dagurval**
Includes https://github.com/bitcoin/bitcoin/pull/7980

### CODE REVIEW

File: qa/rpc-tests/smartfees.py [hunk 1]
```diff
@@ -90,7 +90,8 @@ def split_inputs(from_node, txins, txouts, initial_split = False):
     # If this is the initial split we actually need to sign the transaction
     # Otherwise we just need to insert the property ScriptSig
     if (initial_split) :
-        completetx = from_node.signrawtransaction(rawtx)["hex"]
+        completetx = from_node.signrawtransaction(
+                rawtx, None, None, "ALL|FORKID")["hex"]
```
Comment (dgenr8) [inline qa/rpc-tests/smartfees.py]: This is the default anyway, and the tests generally don't specify the sighashtype unless a non-default value is being used.

Comment (dgenr8) [reply qa/rpc-tests/smartfees.py]: I'm happy to add this, can you just confirm you want it?

Comment (dagurval) [reply qa/rpc-tests/smartfees.py]: I'll check if the first patch fixes the issue by itself. If not, then perhaps there is an issue with defaults.

**Comment by dagurval** [comment]
The first commit fixes the issue by itself. Removed the sighashtype change.

**Pull request merged**